test/unit/nics/bridge_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
+
+class BridgeTest < ActiveSupport::TestCase
+  setup do
+    disable_orchestration
+  end
+
+  test 'is always virtual' do
+    host = FactoryGirl.create(:host)
```

Comment:
Test with `.build`, since the whole purpose of the `def virtual` methods in Nic subclasses seems to be that `.virtual` is true even before validations. After the object is created `ensure_virtual` is created and it'll be virtual for sure.